docs/notebooks/5_warminglevels.ipynb

```diff
@@ -7,6 +7,8 @@
    "source": [
```

Comment:
<p>I think this title should just be somehting like prepare data for tutorial. Or be a higher levels than the titles below.</p><p> </p><p>In general, maybe we can be a bit more clear that we are showing 3 different ways of using the function (wl from name, subset wl and produce horizon), you don't have to do all of them all everytime.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xscen/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>